articles/api-auth/why-use-access-tokens-to-secure-apis.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Why you should always use access tokens to secure an API
+description: Explains the differences between access token and ID token and why the later should never be used to secure an API.
+---
+
+# Why you should always use access tokens to secure an API
+
+There is much confusion on the Web about the differences between the OpenID Connect and OAuth 2.0 specifications, and their respective tokens. As a result many developers publish insecure applications, compromising their users security. The contradicting implementations between identity providers do not help either.
+
+This article is an attempt to clear what is what and explain why you should always use an access token to secure an API, and never an ID token.
+
+## Two complementary specifications
+
+OpenID Connect is used for __user authentication__. It enables you, the user, to verify your identity and give some basic profile information, without sharing your credentials.
```

Comment:
OpendId is a specification that augments OAuth. Oauth only dealt with authorization and doesnt care about how the authorization server verifies who is the user (authentication). this was a problem because home grown solutions for this were created (e.g. facebook connect).
openid augmented oauth by specifying how the AS can authenticate users. as a result, the spec is everything oauth had + authentication (the collateral of this was the introduction of id_tokens).